src/LogExpert/Controls/LogTabWindow/LogTabWindowEventHandlers.cs

```diff
@@ -919,6 +920,12 @@ private void findInExplorerToolStripMenuItem_Click(object sender, EventArgs e)
             explorer.Start();
         }
 
+        private void truncateFileToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            LogWindow logWindow = dockPanel.ActiveContent as LogWindow;
+            File.WriteAllText(logWindow.Title,"");
+        }
+

```

Comment:
Does this take into account, if the file is not write able by another process?
This is not a safe way to create a new file => Exceptions are not taken into account!